.github/workflows/basic.yml

```diff
@@ -92,33 +91,46 @@ jobs:
         compiler: [{c: gcc, cxx: g++}]
         shared_library: ['OFF']
         level_zero_provider: ['ON']
+        install_tbb: ['ON']
         include:
           - os: 'ubuntu-20.04'
             build_type: Release
             compiler: {c: gcc-7, cxx: g++-7}
             shared_library: 'OFF'
             level_zero_provider: 'ON'
+            install_tbb: 'ON'
           - os: 'ubuntu-22.04'
             build_type: Release
             compiler: {c: clang, cxx: clang++}
             shared_library: 'OFF'
             level_zero_provider: 'ON'
+            install_tbb: 'ON'
           - os: 'ubuntu-22.04'
             build_type: Release
             compiler: {c: gcc, cxx: g++}
             shared_library: 'ON'
             level_zero_provider: 'ON'
+            install_tbb: 'ON'
           - os: 'ubuntu-22.04'
             build_type: Debug
             compiler: {c: gcc, cxx: g++}
             shared_library: 'ON'
             level_zero_provider: 'ON'
+            install_tbb: 'ON'
           # test level_zero_provider='OFF'
           - os: 'ubuntu-22.04'
             build_type: Release
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             level_zero_provider: 'OFF'
+            install_tbb: 'ON'
+          # test without installing TBB
```

Comment:
did you verify similar build on Windows and MacOS - I mean check it only one time, not to add extra jobs in CI

Reply:
I verified this on windows locally.